frontend_lib/src/component/Timeline/CommentFilePreview.jsx

```diff
@@ -77,6 +77,19 @@ export const CommentFilePreview = props => {
               buttonIcon='fas fa-ellipsis-v'
               buttonTooltip={props.t('Actions')}
             >
+              {/*
+                FIXME - G.B. - 2021-04-02 - Ideally it would be good to use Link here, but since this
```

Comment:
maybe add a link to an issue there

Reply:
yes, FIXME and TODO comments require a link to a github issue

Reply:
fixed